core/arxiv/submission/services/classifier/classifier.py

```diff
@@ -0,0 +1,144 @@
+"""Classifier service integration."""
```

Comment:
Again, why have this as a separate file from `__init__.py`? `import arxiv.submission.services.classifier.classifier` seems clumsy.